docs/spec/api.md

```diff
@@ -987,7 +987,6 @@ image exists and has been successfully deleted, the following response will be
 issued:
 
     202 Accepted
-    Content-Length: None
```

Comment:
This should actually be.  Though not explicit in the code the empty body has zero length.

`Content-Length: 0`